custom_components/detailed_hello_world_push/sensor.py

```diff
@@ -106,9 +113,7 @@ def unique_id(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes of the device."""
-        attr = {}
-        attr[ATTR_VOLTAGE] = self._roller.battery_voltage
-        return attr
+        return {ATTR_VOLTAGE: self._roller.battery_voltage}
```

Comment:
This is not a good example, since it's a valid significant measurement that should be a separate sensor.

I think we should consider removing `extra_state_attributes` from the example. We really try to avoid using it.

Reply:
Good point, I removed it completely.